bindings/wasm/examples/model-viewer.html

```diff
@@ -110,30 +134,30 @@
   const mv = document.querySelector('model-viewer');
   let objectURL = null;
 
+  const downloadButton = document.querySelector('#download');
+  downloadButton.onclick = function () {
+    const link = document.createElement('a');
+    link.download = 'manifold.glb';
+    link.href = objectURL;
+    link.click();
+  };
+
   async function push2MV(manifold) {
     // From Z-up to Y-up (glTF)
-    const mesh = manifold.rotate([-90, 0, 0]).getMesh();
+    const manifoldMesh = manifold.rotate([-90, 0, 0]).getMesh();
 
-    for (const [i, id] of mesh.runOriginalID.entries()) {
-      const indices = materialMap.get(id);
-      indices.setArray(mesh.triVerts.subarray(mesh.runIndex[i], mesh.runIndex[i + 1]));
+    disposeMesh(node.getMesh());
+    const materials = [];
+    for (const id of manifoldMesh.runOriginalID) {
+      materials.push(materialMap.get(id));
     }
+    const mesh = writeMesh(doc, manifoldMesh, attributes, materials);
+    node.setMesh(mesh);
+    doc.getRoot().listNodes().forEach((n) => console.log(n.listParents()));
+    doc.transform(prune());
+    doc.getRoot().listNodes().forEach((n) => console.log(n.listParents()));
```

Comment:
@donmccurdy Here's the spot where I can see on the command line that two of the three nodes contain only the root as a parent, yet `prune()` doesn't remove them. Any thoughts?

Reply:
Argh, thanks - however interestingly I get exactly the same results. After prune, I have nodes whose only parent is the Root, yet they're not marked as disposed (and prune still says it found nothing to dispose of).

Reply:
Hmm that's a surprise! By default it should remove unused nodes, and also empty leaf nodes. Would it be possible to share the .glb in which prune is failing?